lib/v2/shisu/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    news: ['duuckjing'],
```

Comment:
https://github.com/DIYgod/RSSHub/blob/bf1025e0e3243b870f9909780251b12dd3cbb00c/lib/v2/shisu/router.js#L2
```suggestion
    '/news': ['Duuckjing'],
```